File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Clis/CLIS2015U04GetPatientListItemInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Clis
{
    [Serializable]
    public class CLIS2015U04GetPatientListItemInfo
    {
        private String _protocolPatientId;
        private String _bunho;
        private String _suname;
        private String _suname2;

        public String ProtocolPatientId
        {
            get { return this._protocolPatientId; }
            set { this._protocolPatientId = value; }
        }

        public String Bunho
        {
            get { return this._bunho; }
            set { this._bunho = value; }
        }

        public String Suname
        {
            get { return this._suname; }
            set { this._suname = value; }
        }

        public String Suname2
        {
            get { return this._suname2; }
            set { this._suname2 = value; }
        }

        public CLIS2015U04GetPatientListItemInfo() { }

        public CLIS2015U04GetPatientListItemInfo(String protocolPatientId, String bunho, String suname, String suname2)
        {
            this._protocolPatientId = protocolPatientId;
            this._bunho = bunho;
            this._suname = suname;
            this._suname2 = suname2;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/clis/CLIS2015U04GetPatientListItemInfo.java
/**
 * 
 */
package nta.med.data.model.ihis.clis;

public class CLIS2015U04GetPatientListItemInfo {
	private Integer protocolPatientId;
	private String bunho;
	private String suname;
	private String suname2;

	public CLIS2015U04GetPatientListItemInfo(Integer protocolPatientId,
			String bunho, String suname, String suname2) {
		super();
		this.protocolPatientId = protocolPatientId;
		this.bunho = bunho;
		this.suname = suname;
		this.suname2 = suname2;
	}

	public Integer getprotocolPatientId() {
		return protocolPatientId;
	}

	public void setprotocolPatientId(Integer protocolPatientId) {
		this.protocolPatientId = protocolPatientId;
	}

	public String getBunho() {
		return bunho;
	}

	public void setBunho(String bunho) {
		this.bunho = bunho;
	}

	public String getSuname() {
		return suname;
	}

	public void setSuname(String suname) {
		this.suname = suname;
	}

	public String getSuname2() {
		return suname2;
	}

	public void setSuname2(String suname2) {
		this.suname2 = suname2;
	}
}
